src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -61,6 +61,10 @@ public TestProject([CallerMemberName] string name = null)
 
         public Dictionary<string, string> AdditionalItems { get; } = new Dictionary<string, string>();
 
+        public List<TestItemSpec> AdditionalItemSpecs { get; } = new();
+
+        public string TargetSubDirectory { get; set; }
```

Comment:
It is hard to understand what this is by name

Reply:
Removed.